src/scripts/hooks/render-combat-tracker-config.ts

```diff
@@ -2,7 +2,7 @@ import { htmlClosest, htmlQueryAll } from "@util";
 
 export const RenderCombatTrackerConfig = {
     listen: (): void => {
-        Hooks.on("renderCombatTrackerConfig", async (app, html) => {
+        Hooks.on("renderCombatTrackerConfig", async (_app, html) => {
             // Add "death icon" and "actors dead at zero"
             const appWindow = htmlClosest(html, "#combat-config");
             if (appWindow) appWindow.style.height = "";
```

Comment:
Could you try removing this and see how necessary it is? It's quite a destructive change that we shouldn't make without necessity.